### ISSUE TITLE
production summary remove sheet doesnt work under a folder

### ISSUE THREAD

**Issue by apia46** [shpaass/yafc-ce#79]
v6.1.0, pyanodons ae + text plates
if i make a folder in a summary sheet and put a production sheet in it, i cant remove the production sheet with the remove button
https://github.com/have-fun-was-taken/yafc-ce/assets/96461146/993cf441-e0d3-4315-b382-d564f22ba65d

**Comment by veger** [comment]
The production summary is a partial feature implemented by the original YAFC author. So you are right, it does not have a feature complete implementation...

As we did not implement it, we also do not have a fully vision of this feature and we do not know what is lacking and what is not... It would have been nice to place some warning that this is an incomplete/alpha feature so users are aware that they should use this on their own risk.? (@have-fun-was-taken)

Additionally, I have been working on another production summary view myself in parallel... It has been merged and will be part of the next (0.6.2) release: https://github.com/have-fun-was-taken/yafc-ce/pull/38

It uses a completely different vision than the production summary, so I guess we need to see how/if they can/should  live next to each other. Or whether we need to merge them, or remove one of them.

So for now I doubt that we'll be adding missing features to the production summary until we have some clear idea what to do
